docker/README.md

```diff
@@ -0,0 +1,28 @@
+# Nextcloud Development Environment
+
+## Installation / Running
+
+```bash
+docker-compose up
+```
+
+Afterwards you should be able to open <http://localhost:8081> (admin/admin) to
+log in to your Nextcloud instance.
+
+## Check nextcloud.log
+
+For debugging, you can show the `nextcloud.log`:
+
+```bash
+make show-log
+```
+
```

Comment:
What is the command to execute the tests? I'd like that in the readme

Reply:
There is no npm in the docker image yet.

Reply:
It's just the nextcloud:25-apache image

Reply:
I was thinking about phpunit tests

Reply:
There is no composer neither... Of course you could tinker with the Dockerfile to add all those to the container.

Reply:
I added a how to run test in the readme